src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import task.Task;
+
+/**
+ * Class responsible for adding and deletion of tasks
+ */
+public class TaskList {
+    private List<Task> taskList;
+
+    public TaskList(List<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    public List<Task> getTaskList() {
+        return taskList;
+    }
+
+    public void addTask(Task task) {
+        this.taskList.add(task);
+    }
+
+    /**
+     * Removes task from Arraylist
+     *
+     * @param index index of task to remove.
+     */
+    public void removeTask(int index) {
+        this.taskList.remove(index);
+    }
+
+    public int getSize() {
+        return this.taskList.size();
+    }
+
+    public Task get(int i) {
```

Comment:
Should this method be named "getTask" so as to inform the reader what exactly it is getting? I noticed that other methods here such as <code>removeTask()</code> and <code>getSize()</code> are named in a way that makes it clear to readers what they do, which is great, and perhaps renaming this method would be consistent with this style as well.